### ISSUE TITLE
table has inside div press delete table my div element deleted in TinyMCE

### ISSUE THREAD

**Issue by Valgeria** [tinymce/tinymce#6173]
I am using TinyMCE

I create my editor div element (selector) inside table element. Then I click delete table the div element is deleted.

```
<table>
<tr>
  <td>  <div class="tinymce-body">
<p>Here are some of our customer&rsquo;s most common use cases for TinyMCE:</p>
<ul>
<li>Content Management systems (<em>e.g. WordPress, Umbraco</em>)</li>
<li>Learning Management Systems (<em>e.g. Blackboard</em>)</li>
<li>Customer Relationship Management and marketing automation (<em>e.g. Marketo</em>)</li>
<li>Email marketing (<em>e.g. Constant Contact</em>)</li>
<li>Content creation in SaaS systems (<em>e.g. Eventbrite, Evernote, GoFundMe, Zendesk</em>)</li>
</ul>
<p>And those use cases are just the start. TinyMCE is incredibly flexible, and with hundreds of APIs there&rsquo;s likely a solution for your editor project. If you haven&rsquo;t experienced Tiny Cloud, get started today. You&rsquo;ll even get a free trial of our premium plugins &ndash; no credit card required!</p>  </div>      </td>
</tr>
</table>

<script type="text/javascript">
tinymce.init({
selector: ".tinymce-body",
inline: true,
plugins: 'table',
toolbar: 'table'});
</script>

```
Code: [https://fiddle.tiny.cloud/zhfaab/3](https://fiddle.tiny.cloud/zhfaab/3)

See Image: 
![image](https://user-images.githubusercontent.com/21060460/97952923-ee13c280-1dc4-11eb-957f-295c3c9129ef.png)

How to stop to this. Only allowed to delete 'inside the div element' created tables.

Thanks.

**Comment by ashfordneil** [comment]
Hey, thanks for reporting this. I can reproduce the issue, so I've added it to our internal bug tracker — we'll let you know when a fix is available.

**Comment by obretzke** [comment]
I have the same problem.
I'm using the editor in "inline" mode. The defined editor area is enclosed by a table. If I click in the text area that does not contain a table, the button for deleting a table is still displayed active in the menu. When I click the "delete button", the table that encloses the text field is deleted. So the editor is deleted from the HTML.

**Comment by lnewson** [comment]
Apologies for the late update here, but this was fixed back in TinyMCE 5.8.0. Please let us know if you continue to experience any issues in 5.8.0 or above.
